[2025-02-21T07:29:07.481+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T01:26:00+00:00 [queued]>
[2025-02-21T07:29:07.726+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T01:26:00+00:00 [queued]>
[2025-02-21T07:29:07.739+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-02-21T07:29:09.081+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load_task> on 2025-01-01 01:26:00+00:00
[2025-02-21T07:29:09.156+0000] {standard_task_runner.py:60} INFO - Started process 1586 to run task
[2025-02-21T07:29:09.187+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'optum_etl_dag1', 'load_task', 'scheduled__2025-01-01T01:26:00+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/optum_etl_dag1.py', '--cfg-path', '/tmp/tmp_s6p0qq3']
[2025-02-21T07:29:09.206+0000] {standard_task_runner.py:88} INFO - Job 217: Subtask load_task
[2025-02-21T07:29:09.582+0000] {task_command.py:423} INFO - Running <TaskInstance: optum_etl_dag1.load_task scheduled__2025-01-01T01:26:00+00:00 [running]> on host 491940227afd
[2025-02-21T07:29:10.234+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='optum_etl_dag1' AIRFLOW_CTX_TASK_ID='load_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-01T01:26:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-01T01:26:00+00:00'
[2025-02-21T07:29:10.370+0000] {logging_mixin.py:188} INFO - Loaded data: {'name': 'AIRFLOW', 'type': 'ORCHESTRATION'}
[2025-02-21T07:29:10.382+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-02-21T07:29:10.453+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=optum_etl_dag1, task_id=load_task, execution_date=20250101T012600, start_date=20250221T072907, end_date=20250221T072910
[2025-02-21T07:29:10.702+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-21T07:29:10.962+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
